server/dump.go

```diff
@@ -66,6 +69,11 @@ func (p *Plugin) createDump(ctx context.Context, min, max time.Time, remoteStora
 		return "", err
 	}
 
+	_, err = p.genarateMetadataForDump(filepath.Join(dumpDir, "metadata.yaml"), min, max)
+	if err != nil {
+		return "", err
+	}
+
 	err = zipDirectory(dumpDir, zipFileName)
```

Comment:
Might make sense to define a more specific format for the zip file name so that we can more easily organize them in folders/subfolders, etc. later on - like include type of support package (perf metrics in this case), date it was dumped/collected, customer/company name, whatever else we deem important without needing to open the zip file and read the metadata/other info.

Reply:
Makes sense, would `telemetryID` be enough for a unique identifier? So we can use that for joining on any other information during ingestion. We can rename the support packet name to serverID (aka telemetryID) and I think it would allow us to have bare minimum identification right? Perhaps `telemetryID_timestamp.tar.gz` would even make more sense?

Reply:
I'm thinking smth. like `pm_version_serverId_timestamp.tar.gz`, where 
`pm`- stands for performance metrics (or any other acronym you/your team seem fit)
`version` - stands for plugin version (or maybe we can use the support package version - if we define one and  if we think we will have support package schema changes for the same plugin version. If the support package versions are always tied to a plugin version change, then maybe we can use one as a proxy for the other. If in the future this plugin is used to generate more performance metrics zip files (I believe Maria mentioned more metrics to be Grafana-bound the other day), then it might make sense to have the support package schema version instead (e.g. one for `pm`, one for every other data component generated by the plugin)
 `timestamp` - as above 
 Any opinions: @ifoukarakis ?

Reply:
The name looks fine, as long as it's a separate packet. It's something that can help if we decide to go down the line of having nested. Adding a timestamp can help to uniquely identify the files in case multiple packages are created.

My main concern is that the end-user might decide to rename the file (@nab-77 suggested this happens in the last call). So no matter what we name the file, the same data should be in the manifest file.

Reply:
I agree re: end-users potentially modifying the name. We'll have to see what percentage of them will do that and adjust accordingly. That said, I don't think it should stop us from trying to align support package naming to our wishes.